src/map/pc.c

```diff
@@ -4847,10 +4847,40 @@ int pc_isUseitem(struct map_session_data *sd,int n)
 			if( pc_issit(sd) )
 				return 0;
 			break;
+
+		case ITEMID_GIANT_FLY_WING: {
+			struct party_data *p;
+
+			if (!sd->status.party_id) {
+				clif->msgtable(sd, MSG_PARTY_MEMBER_NOT_SUMMONED);
+				break;
+			}
+
+			if ((p = party->search(sd->status.party_id)) != NULL) {
+				int i;
+				int16 m;
+
+				ARR_FIND(0, MAX_PARTY, i, p->data[i].sd == sd);
+
+				if (i == MAX_PARTY || !p->party.member[i].leader) {
+					clif->msgtable(sd, MSG_PARTY_MEMBER_NOT_SUMMONED);
+					break;
+				}
+
+				m = sd->bl.m;
+
+				ARR_FIND(0, MAX_PARTY, i, p->data[i].sd && p->data[i].sd != sd && p->data[i].sd->bl.m == m);
+
+				if (i == MAX_PARTY || pc_isdead(p->data[i].sd)) {
+					clif->msgtable(sd, MSG_PARTY_NO_MEMBER_IN_MAP);
+					break;
+				}
+			}
+		}
+
```

Comment:
Can you add a `FALLTHROUGH` here? This will prevent static analyzers from marking this as a possible error.